experiments/AMIP/modular/components/ocean/slab_ocean_init.jl

```diff
@@ -157,3 +157,8 @@ Extension of Checkpointer.get_model_state_vector to get the model state.
 function get_model_state_vector(sim::SlabOceanSimulation)
     return sim.integrator.u
 end
+
+get_field(sim::SlabOceanSimulation, ::Val{:energy}) =
+    sim.integrator.p.params.ρ .* sim.integrator.p.params.c .* sim.integrator.u.T_sfc .* sim.integrator.p.params.h
```

Comment:
Could you also explain briefly where this equation comes from (and same for sea ice energy below)?